### PR TITLE
chore: update cross origin command error

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes N/A

### Additional details
Adds explainer to `cross_origin_command` error to point users to `Error Messages` in v12 docs (Link needs to be added). see https://github.com/cypress-io/cypress-documentation/pull/4880
Screen Shot 2022-11-29 at 5 55 44 PM

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/driver/src/cypress/error_messages.ts [hunk 1]
```diff
@@ -913,10 +913,18 @@ export default {
...
+        The command was expected to run against origin \`${commandOrigin}\` but the application is at origin \`${autOrigin}\`.
+
+        This commonly happens when you have either not navigated to the expected origin, have navigated away unexpectedly, or likely forgot to use ${cmd('origin')}.
 
-        This commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.`
+        \`cy.origin('${autOrigin}', () => {\`
+        \`  <commands targeting ${autOrigin} go here>\`
+        \`})\`
+        `,
+        docsUrl: 'https://on.cypress.io/cy-visit-succeeded-but-commands-are-timing-out-because-you-visited-a-different-origin-domain-without-using-cy-origin',
```
Comment (emilyrohrbough) [inline packages/driver/src/cypress/error_messages.ts]: whoa....this feels crazy verbose. Any way we could shorten?
```
https://on.cypress.io/visit-failed-wrong-origin',
```

Comment (AtofStryker) [reply packages/driver/src/cypress/error_messages.ts]: I think we can I just have no idea how these mappings are generated or linked. Thats just the header link in https://github.com/cypress-io/cypress-documentation/pull/4880

Comment (emilyrohrbough) [reply packages/driver/src/cypress/error_messages.ts]: We need to create a `on` link in the services repo with the docs reference

Comment (AtofStryker) [reply packages/driver/src/cypress/error_messages.ts]: hows `https://on.cypress.io/cy-visit-succeeded-but-commands-fail`?

Comment (AtofStryker) [reply packages/driver/src/cypress/error_messages.ts]: got a link branch updated with the changes https://github.com/cypress-io/cypress-services/compare/chore/update-error-message-mapping?expand=1

File: packages/driver/src/cypress/error_messages.ts [hunk 2]
```diff
@@ -913,10 +913,18 @@ export default {
     },
     test_stopped: 'Cypress test was stopped while running this command.',
     cross_origin_command ({ commandOrigin, autOrigin }) {
-      return stripIndent`\
-        The command was expected to run against origin \`${commandOrigin }\` but the application is at origin \`${autOrigin}\`.
+      return {
+        message: stripIndent`\
+        The command was expected to run against origin \`${commandOrigin}\` but the application is at origin \`${autOrigin}\`.
+
+        This commonly happens when you have either not navigated to the expected origin, have navigated away unexpectedly, or likely forgot to use ${cmd('origin')}.
```
Comment (chrisbreiding) [inline packages/driver/src/cypress/error_messages.ts]:
```suggestion
        This commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly. 
        
        Using ${cmd('origin')} to wrap the commands run on \`${autOrigin}\` will likely fix this issue.
```

**Review by chrisbreiding — CHANGES_REQUESTED** [review]

File: packages/driver/src/cypress/error_messages.ts (hunk 2 above)
Comment (AtofStryker) [reply packages/driver/src/cypress/error_messages.ts]: addressed in 12d5ce1f19

**Review by emilyrohrbough — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Review by chrisbreiding — APPROVED** [review]

**Comment by AtofStryker** [comment]
on link PR update: https://github.com/cypress-io/cypress-services/pull/5040

**Pull request merged**